CHANGELOG.md

```diff
@@ -29,6 +29,11 @@
 * Prevent crash for private types named `_` in `type_name` rules.
   [sinoru](https://github.com/sinoru)
   [#3971](https://github.com/realm/SwiftLint/issues/3971)
+  
+* Make `for_where` rule implementation independent of order in structure
```

Comment:
can you please add that it fixes this rule on Xcode 13.3?

Reply:
Good point. Done.